generator/lib/src/realm_model_info.dart

```diff
@@ -34,41 +36,51 @@ class RealmModelInfo {
   Iterable<String> toCode() sync* {
     yield 'class $name extends $modelName with RealmEntity, RealmObjectBase, ${baseType.className} {';
     {
-      final allSettable = fields.where((f) => !f.type.isRealmCollection && !f.isRealmBacklink).toList();
+      final allSettableFields = fields.where((f) => !f.type.isRealmCollection && !f.isRealmBacklink).toList();
```

Comment:
Not a fan of this name change - repeating the type in the name

Reply:
will revert